pkgs/tools/networking/hysteria/default.nix

```diff
@@ -4,16 +4,16 @@
 }:
 buildGoModule rec {
   pname = "hysteria";
-  version = "2.2.2";
+  version = "2.2.3";
 
   src = fetchFromGitHub {
     owner = "apernet";
     repo = pname;
     rev = "app/v${version}";
-    hash = "sha256-5j24wIZ4LloE9t0sv5p+oiYmexOaORASNN9JylXxrk4=";
+    hash = "sha256-xvnshGDQ69yXZ5BnEYAhoJOXG0uZ0lZRnp/rdnmNAkE=";
   };
 
-  vendorHash = "sha256-ErU1yEtSuMVkoJv9hyaE4OZS5o7GxuleoK0Q9BI2skw=";
+  vendorHash = "sha256-zjj5MhCVQbsLwZs7LcGhozXGmfzFrDipNsEswiseMYI=";
   proxyVendor = true;
 
   ldflags = [
```

Comment:
The `hysteria version` output does not seem to provide the right version.

IMHO, maybe we might need check again `-X main.appVersion=${version}` and probably replace it with `-X cmd.appVersion`.

Reply:
> The `hysteria version` output does not seem to provide the right version.
> 
> IMHO, maybe we might need check again `-X main.appVersion=${version}` and probably replace it with `-X cmd.appVersion`.

Thanks!